.gitban/templates/spike-project-closeout.md

```
@@ -0,0 +1,257 @@
+
+   ```bash
+   # Use archive_cards tool to create sprint retrospective
+   archive_cards(
+       archive_name="project-auth-v2-complete",
+       all_done=True,  # Archive all completed cards for this project
+       preview=False   # Actually execute the archive
+   )
+   ```
+
+4. **Generate Sprint Summary:**
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)